src/transport.ts

```diff
@@ -70,24 +70,23 @@ export default class SentryTransport extends TransportStream {
 
     const sentryLevel = this.levelsMap[winstonLevel];
 
-    Sentry.configureScope((scope) => {
-      scope.clear();
+    const scope = Sentry.getCurrentScope();
+    scope.clear();
```

Comment:
Is this needed? in migration docs it seems like it suggest is to get current scope only without clear?

```diff
- Sentry.configureScope((scope) => {
-  scope.setTag("key", "value");
- });
+ Sentry.getCurrentScope().setTag("key", "value");
```

[-of-sentryconfigurescope-method](https://docs.sentry.io/platforms/javascript/migration/v7-to-v8/#removal-of-sentryconfigurescope-method)